app/screens/ChildSetup.tsx

```diff
@@ -252,7 +252,7 @@ const ChildSetup = ({ navigation }: Props): any => {
       if (dataset.name.endsWith('.json')) {
         const decryptedData = decryptData(dataset.data, encryptionsKey)
           .then((text: any) => {
-            return text;
+            return text.replace(/[\x00-\x1F\x7F]/g,'');

```

Comment:
We apply this for each file in backup file and hope it does not effect the performance.

Reply:
No impact